frontend/src/Settings/index.jsx

```diff
@@ -131,41 +141,34 @@ function Settings() {
       <View style={styles.container}>
         <Text style={styles.titleText}>Settings</Text>
 
-        <Text style={styles.labelText}>Email</Text>
         <InputField
-          placeholder={useSelector((state) => state.settings.email)}
-          value={emailInput}
-          onChangeText={(value) => setEmailInput(value)}
+          placeholder="Username"
+          value={username}
+          onChangeText={(value) => setUsername(value)}
         />
 
-        <Text style={styles.labelText}>Username</Text>
         <InputField
-          placeholder={useSelector((state) => state.settings.username)}
-          value={usernameInput}
-          onChangeText={(value) => setUsernameInput(value)}
+          placeholder="Email"
+          value={email}
+          onChangeText={(value) => setEmail(value)}
         />
 
-        <Text style={styles.labelText}>Enter your new Kingdom Name here</Text>
         <InputField
-          placeholder={useSelector((state) => state.settings.kingdomName)}
-          value={kingdomNameInput}
-          onChangeText={(value) => setKingdomNameInput(value)}
+          placeholder="Password"
+          value={password}
+          onChangeText={(value) => setPassword(value)}
+          secureTextEntry
         />
 
-        <Text style={styles.labelText}>Password</Text>
         <InputField
-          placeholder="password123"
-          value={passwordInput}
-          onChangeText={(value) => setPasswordInput(value)}
-          secureTextEntry
+          placeholder="Kingdom Name"
+          value={kingdomName}
+          onChangeText={(value) => setKingdomName(value)}
         />
 
         <View style={styles.buttonRow}>
+          <SubmitButton onPress={handleLogout} text="Logout" />
           <SubmitButton onPress={handleSubmit} disabled={submitButtonIsDisabled} text="Save" />
```

Comment:
please keep the cancel button and change the order of cancel and save button, save is always on the right, cancel is always on the left, and because of this I accidentally clicked on Logout 3 times in a row :D